src/mlpack/core/optimizers/nestrov/nag_impl.hpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file nag_impl.hpp
+ * @author Ryan Curtin
+ * @author Kris Singh
+ *
+ * Implementation of stochastic gradient descent.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_NAG_IMPL_HPP
```

Comment:
The header guard should be named like: MLPACK_CORE_OPTIMIZERS_NESTROV_NAG_IMPL_HPP.